vertx-sql-client/src/main/java/io/vertx/sqlclient/Pool.java

```diff
@@ -152,6 +153,14 @@ static Pool pool(Vertx vertx, SqlConnectOptions connectOptions, PoolOptions pool
    */
   <T> Future<T> withTransaction(Function<SqlClient, Future<T>> function);
 
+  @GenIgnore
+  // FIXME codegen
```

Comment:
what is the codegen error here ?

Reply:
it should be ok, I just don't add codegen annotations for a POC prototype, will add it later